convert_tendl_rand.py

```diff
@@ -0,0 +1,360 @@
+#https://tendl.web.psi.ch/tendl_2017/tar_files/Na23.random.tgz  130MB   831     
+#https://tendl.web.psi.ch/tendl_2017/tar_files/Si28.random.tgz  110MB   500     x
+#https://tendl.web.psi.ch/tendl_2017/tar_files/Si29.random.tgz  100MB   500     x
+#https://tendl.web.psi.ch/tendl_2017/tar_files/Si30.random.tgz  80MB    500     x
+#https://tendl.web.psi.ch/tendl_2017/tar_files/Fe054.random.tgz 320MB   500     x
+#https://tendl.web.psi.ch/tendl_2017/tar_files/Fe056.random.tgz 410MB   613     x
+#https://tendl.web.psi.ch/tendl_2017/tar_files/Fe057.random.tgz 525MB   732     x
+#https://tendl.web.psi.ch/tendl_2017/tar_files/Fe058.random.tgz 280MB   500     x
+#https://tendl.web.psi.ch/tendl_2017/tar_files/Ge.random.tgz    1.3GB   ?   
+#https://tendl.web.psi.ch/tendl_2017/tar_files/Pu240.random.tgz 200MB   624 
+#https://tendl.web.psi.ch/tendl_2015/tar_files/Al27.random.tgz  150MB   300     
+#https://tendl.web.psi.ch/tendl_2015/tar_files/Si28.random.tgz  125MB   ...     x
+#https://tendl.web.psi.ch/tendl_2015/tar_files/Ti48.random.tgz  140MB    ↓
+#https://tendl.web.psi.ch/tendl_2015/tar_files/Ni58.random.tgz  210MB           x
+#https://tendl.web.psi.ch/tendl_2015/tar_files/Cu63.random.tgz  205MB           
+#https://tendl.web.psi.ch/tendl_2015/tar_files/Cu65.random.tgz  180MB
+#https://tendl.web.psi.ch/tendl_2015/tar_files/Zr90.random.tgz  150MB
+#https://tendl.web.psi.ch/tendl_2015/tar_files/Zr91.random.tgz  175MB
+#https://tendl.web.psi.ch/tendl_2015/tar_files/Zr92.random.tgz  170MB
+#https://tendl.web.psi.ch/tendl_2015/tar_files/W180.random.tgz  210MB
+#https://tendl.web.psi.ch/tendl_2015/tar_files/W182.random.tgz  215MB
+#https://tendl.web.psi.ch/tendl_2015/tar_files/W183.random.tgz  215MB
+#https://tendl.web.psi.ch/tendl_2015/tar_files/W184.random.tgz  250MB   ...
+#https://tendl.web.psi.ch/tendl_2015/tar_files/W186.random.tgz  180MB   300
+
+#RandomAce
+
+#https://tendl.web.psi.ch/tendl_2015/tar_files/H1.nuss.05.10.2016.tgz   11MB    340
+#https://tendl.web.psi.ch/tendl_2015/tar_files/O16.nuss.05.10.2016.tgz  570MB   340
+#https://tendl.web.psi.ch/tendl_2015/tar_files/U235.nuss.10.10.2016.tgz 430MB   100
+#https://tendl.web.psi.ch/tendl_2015/tar_files/U238.nuss.10.10.2016.tgz 1.5GB   100
+
+#ParsedNuclides._get_args('nuclides')
+
+if( 'all' in args.nuclides):
+    list_= ['O16','Si28', 'Si29','Si30', 'Fe54', 'Fe56', 'Fe57', 'Fe58']
+else:
+    list_ = args.nuclides
+scriptDir = os.getcwd()
+
+library_name = 'tendl_rand' #this could be added as an argument to allow different libraries to be downloaded
+
+
+# the destination is decided after the release is know to avoid putting the release in a folder with a misleading name
+if args.destination is None:
+    outputDir = os.path.abspath(os.getcwd() + '/' + library_name)
+else:
+    outputDir = os.path.abspath(args.destination)
+
+
+endf_files_dir  = '/'.join([outputDir, 'endf'])
+ace_files_dir   = '/'.join([outputDir, 'ace'])
+hdf5_files_dir  = '/'.join([outputDir, 'hdf5'])
```

Comment:
In other scripts, we've adopted use of the [pathlib](https://docs.python.org/3/library/pathlib.html) standard library module for handling file/directory names. I'd recommend going through your script and using pathlib throughout